synapse/models/person.py

```diff
@@ -178,6 +219,18 @@ def getBaseModels():
                     ('seen:min', {'ptype': 'time:min'}),
                     ('seen:max', {'ptype': 'time:max'}),
                 )),
+
+                ('ps:has', {}, [
+                    ('person', {'ptype': 'ps:person', 'ro': 1, 'req': 1,
+                        'doc': 'The person that has the given node.'}),
+                    ('xref', {'ptype': 'propvalu', 'ro': 1, 'req': 1,
+                        'doc': 'The prop=valu that is referenced as part of the FIXME.'}),
```

Comment:
Holding off on updating these docstrings until the end. Will get input from @therealsilence

Reply:
Suggested: "The object or resource (prop=valu) that is owned or controlled by the person."

Reply:
fixed